Speak-like-Yoda/Speak_Like_yoda.py

```diff
@@ -0,0 +1,24 @@
+import random
+import string
+
+def speak_like_yoda(sentence):
+    """
+    Translate the input sentence into Yoda-speak.
+    
+    :param sentence: input string
+    :return: translation to Yoda-speak
+    """
+    sentence = sentence.lower()
+    for p in string.punctuation.replace("'", ''):
+        sentence = sentence.replace(p, '')
+    words = sentence.split()
+    random.shuffle(words) 
+    new_sent = ' '.join(words)
+    print()
```

Comment:
why empty print statement?

Reply:
```
Your English sentence:
Avoiding failure is something we learn at some later point in life.
                      #this line for print()
Your Yodenglish sentence:
Something is we point some failure learn avoiding later at in life
```

Reply:
Use `\n` in starting of  `print('Your Yodenglish sentence: ')`